src/tribler-core/tribler_core/components/version_check.py

```diff
@@ -8,6 +8,8 @@ class VersionCheckComponent(Component):
     version_check_manager: VersionCheckManager
 
     async def run(self):
```

Comment:
**nit**: it doesn't relate to the PR, but it seems that `await super().run()` is missed

Reply:
Fixed